tools/hardklor/hardklor.py

```diff
@@ -13,7 +13,7 @@
     'mzml': 'mzML',
     'mzxml': 'mzXML',
     'ms1': 'MS1',
-    }
+}
```

Comment:
I don't understand this diff ...
I fixed that already in master: https://github.com/galaxyproteomics/tools-galaxyp/blob/master/tools/hardklor/hardklor.py

Why is github showing this diff?

Reply:
Me neither.  Lets just close it,